src/unix/bsd/freebsdlike/freebsd/mod.rs

```diff
@@ -4957,6 +5028,24 @@ f! {
     pub fn PROT_MAX_EXTRACT(x: c_int) -> c_int {
         (x >> 16) & (crate::PROT_READ | crate::PROT_WRITE | crate::PROT_EXEC)
     }
+
+    pub {const} fn ETHER_IS_MULTICAST(addr: *mut u_char) -> bool {
+        (*addr.wrapping_add(0)) & 0x01 != 0x00
```

Comment:
I'm surprised that Clippy doesn't complain about the uselessness of `wrapping_add(0)`.